.github/workflows/bls.yml

```diff
@@ -1,17 +0,0 @@
-on: [pull_request]
```

Comment:
Why this workflow is deleted. We want to make sure Zarb can be compiled with bls library.

Reply:
> it might be better idea to abandon compiling bls repo and use the commit above as nested repo or maybe use it only for releases and keep compiling for docker and development

as i suggested in the pull request  comment i deleted it to show case abandoning it